mods/Fifty.ServerUtilities/mod/scripts/vscripts/fsu_core.nut

```diff
@@ -95,9 +99,9 @@ void function FSU_ChatBroadcast( string message, bool usePopUp = false ) {
 			NSSendPopUpMessageToPlayer( player, message )
 	} else {
 		if( GetConVarBool( "FSU_PREFIX_SYSTEM_MESSAGES" ) )
-			Chat_ServerBroadcast( FSU_FormatString( "%F[FSU]%T " + message ), false )
+			Chat_ServerBroadcast( FSU_FormatString( "%F[FSU]%N " + message ), false )

```

Comment:
![image](https://user-images.githubusercontent.com/64418963/214290459-0ea9bde8-2485-4c19-83f3-d2c85ac349f7.png)
Might be worth editing the convars to match this change.

Reply:
yes